unittests/test_fields.py

```diff
@@ -669,6 +671,22 @@ def test_delitem(self):
             KeyError, exec, "del scoped_dict['a:k4']", globals(), locals()
         )
 
+    def test_delparent_scope(self):
```

Comment:
This name does not reflect the purpose of this test. The purpose of the test is "Adding a scope with a same name as a key"

Reply:
By nested scope do you mean the `s0:s1` part of `s0:s1:k0`?

Reply:
No, this is not nested! In `s0:s1`, `s1` is a key, not a scope. In the second one `s0` and `s1` are scopes. This test checks what happens if a key and scope have the same name. This test is valid (with the adjustments I proposed) and you should just change its name. The nested scopes are different a different thing and should be in a different test function. Here is an example:

```python
s = ScopedDict()
s['s0:k0'] = 1
s['s0:s1:k0'] = 2
s['*:k0'] = 3
del s['s0']
assert s['s0:k0'] == 3
assert s['s0:s1:k0'] == 2
```